object/object.go

```diff
@@ -67,50 +67,52 @@ func NativeBoolToBooleanObject(input bool) Boolean {
 	return FALSE
 }
 
-func Equals(left, right Object) Object {
+func Equals(left, right Object) bool {
 	if left.Type() != right.Type() {
-		return FALSE
+		return false
 	}
 	switch left := left.(type) {
 	case Integer:
-		return NativeBoolToBooleanObject(left.Value == right.(Integer).Value)
```

Comment:
Are you still using NativeBoolToBooleanObject func?

If not maybe, it can be deleted

Reply:
Same question for TRUE and FALSE objects

Reply:
see the rest of the diff... yes it's needed because inside grol that's the objects for `true` and `false`